### PR TITLE
[propose] add checklist and small tweaks

### PR DESCRIPTION
**Pull request by cpanato**
#### What this PR does / why we need it:
- Add checklist in the PR to help the users to fill a PR
- Small tweaks

#### Which issue this PR fixes
 - n/a

#### Special notes for your reviewer:
 - Using the proposed new template

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed

cc @mattfarina @unguiculus @davidkarlsen

### CODE REVIEW

**Comment by cpanato** [comment]
ping @unguiculus @mattfarina

**Comment by davidkarlsen** [comment]
I think this looks good to make people more aware of the basics.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -25,8 +25,16 @@ very quickly. Please check the results. We would like these checks to pass befor
...
+    - fixes #
 
-**Special notes for your reviewer**:
+#### Special notes for your reviewer:
+
+#### Checklist
+[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
+- [ ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
+- [ ] Chart Version bumped
+- [ ] Variables are in the README.md
```
Comment (davidkarlsen) [inline .github/PULL_REQUEST_TEMPLATE.md]: Nit-picking - Variables are documented in the README.md

**Pull request merged**